docs/development/guidelines.md

```diff
@@ -370,7 +429,43 @@ Examples
 
 ## Tests
```

Comment:
It might be worth splitting this into some more subsections, including:

* File structure (with do/don't example?)
* Naming (with do/don't example?)
* Parametrization (exists already)

Reply:
Splitted it into the suggested subsections and added a do / don't example for naming.

For the file structure, I'm not sure how such an example should look like. We cannot show Python code as in all the other examples.
At the same time, it should be fairly clear to understand, so I'm not sure examples are necessary here?

Reply:
It's fine as it's now.